Generator/BotBuilderVSIX-V4/ProjectTemplates/Bot Framework/TestBot.bot

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What's this file?

Reply:
This file contains the user information. It is parsed by the emulator. I believe all new bots will contain this file.

Reply:
Ah. Is there a way to add a comment in here?

Reply:
Seems that we don't need this file after all. I will get rid of it.